### PR TITLE
Dev 11406 listdatabasedownload  [hotfix][qat]

### PR DESCRIPTION
**Pull request by LKambhampati-frb**
Duplicate of #4181

### CODE REVIEW

**Comment by aguest-kc** [comment]
🛑 Don't merge QAT into the new branch until the Staging and Master PRs have been merged. 🛑

**Review by collinwr — APPROVED** [review]

**Pull request merged**
